pkg/cloudprovider/providers/openstack/openstack_volumes.go

```diff
@@ -288,6 +320,32 @@ func (os *OpenStack) DetachDisk(instanceID, volumeID string) error {
 	return nil
 }
 
+// ExpandVolume expands the size of specific cinder volume (in GiB)
+func (os *OpenStack) ExpandVolume(volumeID string, newSize int) error {
+	volume, err := os.getVolume(volumeID)
+	if err != nil {
+		return err
+	}
+	if volume.Status != VolumeAvailableStatus {
+		// cinder volume can not be expanded if its status is not available
+		glog.V(2).Infof("the status of volume: %s is not available", volume.ID)
+		return fmt.Errorf("volume status is not available")
+	}
+
+	volumes, err := os.volumeService("")
+	if err != nil || volumes == nil {
+		glog.Errorf("Unable to initialize cinder client for region: %s", os.region)
```

Comment:
You know what I'm going to say at this point ;)  Also, this error doesn't actually include `err`.